extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -107,47 +107,48 @@ describe('Project: sqlproj content operations', function (): void {
 		sinon.restore();
 	});
 
-	it('Should add Folder and Build entries to sqlproj', async function (): Promise<void> {
-		const project = await Project.openProject(projFilePath);
+	it('Should perform Folder and SQL object script operations', async function (): Promise<void> {
+		const project = await testUtils.createTestSqlProject();
 
-		const folderPath = 'Stored Procedures\\';
+		const folderPath = 'Stored Procedures';
 		const scriptPath = path.join(folderPath, 'Fake Stored Proc.sql');
 		const scriptContents = 'SELECT \'This is not actually a stored procedure.\'';
 
 		const scriptPathTagged = path.join(folderPath, 'Fake External Streaming Job.sql');
 		const scriptContentsTagged = 'EXEC sys.sp_create_streaming_job \'job\', \'SELECT 7\'';
 
+		(project.folders.length).should.equal(0);
+		(project.files.length).should.equal(0);
+
 		await project.addFolder(folderPath);
 		await project.addScriptItem(scriptPath, scriptContents);
 		await project.addScriptItem(scriptPathTagged, scriptContentsTagged, ItemType.externalStreamingJob);
 
-		const newProject = await Project.openProject(projFilePath);
-
-		should(newProject.files.find(f => f.type === EntryType.Folder && f.relativePath === convertSlashesForSqlProj(folderPath))).not.equal(undefined);
-		should(newProject.files.find(f => f.type === EntryType.File && f.relativePath === convertSlashesForSqlProj(scriptPath))).not.equal(undefined);
-		should(newProject.files.find(f => f.type === EntryType.File && f.relativePath === convertSlashesForSqlProj(scriptPathTagged))).not.equal(undefined);
-		should(newProject.files.find(f => f.type === EntryType.File && f.relativePath === convertSlashesForSqlProj(scriptPathTagged))?.sqlObjectType).equal(constants.ExternalStreamingJob);
-
-		const newScriptContents = (await fs.readFile(path.join(newProject.projectFolderPath, scriptPath))).toString();
+		(project.folders.length).should.equal(1);
+		(project.files.length).should.equal(2);
 
-		should(newScriptContents).equal(scriptContents);
+		should(project.folders.find(f => f.relativePath === convertSlashesForSqlProj(folderPath))).not.equal(undefined);
+		should(project.files.find(f => f.relativePath === convertSlashesForSqlProj(scriptPath))).not.equal(undefined);
+		should(project.files.find(f => f.relativePath === convertSlashesForSqlProj(scriptPathTagged))).not.equal(undefined);
+		// TODO: support for tagged entries not supported in DacFx.Projects
+		//should(project.files.find(f => f.relativePath === convertSlashesForSqlProj(scriptPathTagged))?.sqlObjectType).equal(constants.ExternalStreamingJob);
 	});
 
-	it('Should add Folder and Build entries to sqlproj with pre-existing scripts on disk', async function (): Promise<void> {
-		projFilePath = await testUtils.createTestSqlProjFile(baselines.newProjectFileBaseline);
-		const project = await Project.openProject(projFilePath);
+	// TODO: addToProject() can probably be removed
```

Comment:
this was being used by the oracle migration extension, so it would probably break their extension if `addToProject()` is removed without letting them know

Reply:
Sergey Tan is the point of contact over there, right?